stubs/tensorflow/tensorflow/keras/layers/preprocessing/index_lookup.pyi

```diff
@@ -0,0 +1,9 @@
+from _typeshed import Incomplete
+
+import tensorflow as tf
+from tensorflow.keras.layers.experimental.preprocessing import PreprocessingLayer
+
+class IndexLookup(PreprocessingLayer):
```

Comment:
This class exists at runtime and is helpful for IntegerLookup/StringLookup, but it's undocumented internal class. So I'd lean to underscore prefix it here.

Reply:
Done in 31134a1b1.